src/main/java/io/codelex/arrays/practice/Hangman.java

```diff
@@ -0,0 +1,62 @@
+package io.codelex.arrays.practice;
+
+import java.util.Scanner;
+
+public class Hangman {
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String[] listOfWords = {"building", "river", "road"};
+        String guessedLetters = "";
+        String missedLetters ="";
+        String chooseWord = listOfWords[(int) (Math.random() * listOfWords.length)];
+
+        while(true){
+            System.out.println("-=-=-=-=-=-=-=-=-=-=-=-=");
+            System.out.println();
+            System.out.println("Word: ");
+            printOutWord(chooseWord, guessedLetters);
+            System.out.println();
+            System.out.println("Misses:");
+            System.out.println(missedLetters);
+            if(missedLetters.length()>=10){
+                System.out.println("You loose the game");
+            }
+
+            System.out.println();
+            System.out.println("Guess: ");
+
+            char inputChar = scan.nextLine().charAt(0);
+
+            //check if chosen word contains input letter, if not then input char pushed to missed letters
+            if( chooseWord.contains(String.valueOf(inputChar))) {
+                guessedLetters += inputChar;
+            } else {
+                missedLetters +=inputChar;
+            }
+
+            //check if you guessed all characters
+            if(guessedLetters.equals(chooseWord)){
+                System.out.println("You won the game");
+                System.out.println("Guessed word is: ");
+                printOutWord(chooseWord, guessedLetters);
+                break;
+            }
+
+
+        }
+
+
```

Comment:
formatting: blank lines līdz 1 max labāk in general

Reply:
Izdzēsu tukšās līnijas